### PR TITLE
Add .dependabot/config.yml

### PR DESCRIPTION
**Pull request by gastaldi**
According to the discussion in https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/quarkus-dev/7KAjUz5iti0/YzGCrCN7BwAJ

This must be merged before Dependabot is enabled in this repository

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - package_manager: "java:maven"
+    directory: "/bom/runtime"
+    update_schedule: "daily"
+    allowed_updates:
+      - match:
+          update_type: "security"
+      - match:
+          dependency_name: "org.flywaydb:flyway-core"
+      - match:
+          dependency_name: "org.eclipse.jgit:org.eclipse.jgit"
```
Comment (gsmet) [inline .dependabot/config.yml]: With this config, it will just create PRs?

Comment (gastaldi) [reply .dependabot/config.yml]: Yes, automerged updates must be enabled at the account level (from account settings in your dashboard) before they can be configured on a project.

https://dependabot.com/docs/config-file/#automerged_updates

**Comment by gastaldi** [comment]
Also, when enabling Dependabot to the Quarkusio org, make sure to disable automatic rebase, because CI

https://github.com/dependabot/feedback/issues/152#issuecomment-414800553

**Comment by gsmet** [comment]
Let's wait until next week to see if someone has a strong argument against it.

Then I'll merge and could we sync to do a hangout with screen share then so that I properly enable it on the repo and set all the appropriate option?

**Comment by gastaldi** [comment]
Sure, that works for me. Thanks!

**Comment by geoand** [comment]
Makes sense to start small and add more if needed. Maybe we should add the Kubernetes Client @gastaldi ?

**Comment by gastaldi** [comment]
> Makes sense to start small and add more if needed. 

@geoand definitely, that's the idea :)

> Maybe we should add the Kubernetes Client @gastaldi ?

Good idea, let me update it

**Comment by cescoffier** [comment]
Adding the WIP prefix just to make sure it's not merged before the discussion completes.

**Comment by gsmet** [comment]
Not entirely sure about the security thing (mostly because of Jackson and the Vert.x dependency) but let's see how it goes.

**Review by gsmet — APPROVED** [review]

**Pull request merged**